### DISCUSSION TITLE
What's best ReactTable or useTable or why don't I see much use of ReactTable

### DISCUSSION THREAD

**Discussion by erikswed** [TanStack/table#3164] [Q&A]
I learn Javascript and React and reading about react-table trying to learn it.

I came across this[ codesandbox that uses import ReactTable from "react-table"](https://codesandbox.io/s/react-table-simple-table-forked-14bmv?file=/index.js:141-151) and when reading the [docs I cant find any example of this ReactTable import](https://react-table.tanstack.com/docs/examples/basic)

Is this `ReactTable `about to be deprecated maybe? I see the hook style `useTable` looks more versatile 

Any thought?

**Comment by salmanbabri** [comment]
Before v7 was released, react-table used to provide UI components as you can see in that code sandbox. 

V7 onwards, however, react-table has been changed to a headless table utility library, meaning that you'll need to provide the UI yourself while react-table will help you with populating the table with data & applying sorting, pagination etc.

In summary, follow the documentation & start by using `useTable` hook to create a basic table. Hope that helps.

**Reply by erikswed** [reply]
Thanks I did not know that I suspected it thanks for clarifying

**Reply by Agent-Gautam** [reply]
Thanks for this
